packages/EasyBatch/tests/Bridge/Symfony/AbstractSymfonyTestCase.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyBatch\Tests\Bridge\Symfony;
+
+use EonX\EasyBatch\Tests\AbstractTestCase;
+use EonX\EasyBatch\Tests\Bridge\Symfony\Stubs\KernelStub;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+abstract class AbstractSymfonyTestCase extends AbstractTestCase
+{
+    /**
+     * @var \Symfony\Component\HttpKernel\KernelInterface
```

Comment:
`|null` since we have comparing with `null` in 20 line?

Reply:
Done